src/presets/deno-deploy.ts

```diff
@@ -19,7 +19,9 @@ export const denoDeploy = defineNitroPreset({
     external: ["https://deno.land/std/http/server.ts"],
     output: {
       entryFileNames: "index.ts",
-      manualChunks: () => "index",
+      manualChunks: (id) => {
+        if (id !== "https://deno.land/std/http/server.ts") { return "index"; }
```

Comment:
Would it make sense that we check if id starting with http(s) protocol?

Reply:
Probably. There is an upstream fix coming though, so I'm not sure if this PR makes sense: https://github.com/rollup/rollup/issues/5067#issuecomment-1636827568, maybe it's better to wait for upstream.

Reply:
Actually, it wouldn't, because only `"https://deno.land/std/http/server.ts"` is external right now. But that could probably also be fixed, imports from URLs probably fail right now...

Reply:
I agree we should probably match id that start with `https://`. Let's wait for https://github.com/rollup/rollup/pull/5068